pkg/db/db.go

```diff
@@ -445,18 +448,32 @@ type DBReleaseMetaData struct {
 	DisplayVersion string
 }
 
+type DBReleaseManifests struct {
+	Manifests map[string]string
+}
+
 type DBReleaseWithMetaData struct {
 	EslId         EslId
 	ReleaseNumber uint64
+	Created       time.Time
 	App           string
-	Env           string
-	Manifest      string
+	Manifests     DBReleaseManifests
 	Metadata      DBReleaseMetaData
 }
 
+type DBAllReleaseMetaData struct {
```

Comment:
I find it hard to understand this notion of "metadata". Maybe we should rename this? If not, let's add an explanatory comment for why "metadata" isn't used traditionally here.

Reply:
I didn't come up with a good alternative name.
I would say let's do this in another PR.